src/main/java/org/fcrepo/camel/FcrepoClient.java

```diff
@@ -81,7 +82,7 @@ public FcrepoResponse head(final URI url)
         final HttpRequestBase request = HttpMethods.HEAD.createRequest(url);
         final HttpResponse response = executeRequest(request);
         final int status = response.getStatusLine().getStatusCode();
-        final String contentType = getContentTypeHeader(response);
+        final Optional<String> maybeContentType = getContentTypeHeader(response);
```

Comment:
@ajs6f this is precisely the feedback I'm looking for. It's a bit unclear to me when `Optional` should be used, if at all in this code. That is, I'm not at all attached to the refactoring in this PR -- it is more an exercise in getting to know the `Optional` class.